fixtures/sample-project-with-scss/src/sample-collection/sample-fragment/fragment.json

```diff
@@ -0,0 +1,10 @@
+{
+  "cssPath": "styles.scss",
+  "configurationPath": "configuration.json",
+  "htmlPath": "index.html",
+  "jsPath": "main.js",
+  "sass": {
```

Comment:
Are these values kept after importing and exporting to the portal?

Reply:
Nope, everything is removed, it's imported as a regular fragment (we need to warn about this in the documentation).

Reply:
Agree, is there a documentation created for that?

Reply:
Nope, I have to create a documentation draft inside the story. I just added it so I don't forget. https://issues.liferay.com/browse/LPS-147256